sys-libs/musl/musl-1.2.2-r8.ebuild

```diff
@@ -89,6 +89,7 @@ src_prepare() {
 
 src_configure() {
 	filter-lto # bug #877343
+	replace-flags -Ofast -O3 # Ofast causes issues with Musl
```

Comment:
Given it's a libc, it'd be appropriate to use strip-flags here tbh.

Reply:
I just made this change now and removed the filter-lto as it does the same task.